build/project-template/internal/nativescript-pre-build

```diff
@@ -1,6 +1,2 @@
 #!/usr/bin/env bash
-set -e
-
-pushd "$SRCROOT/internal/metadata-generator/bin"
-./build-step-metadata-generator.py
-popd
+export >./env_vars.sh
```

Comment:
Which directory will be the current working dir. Shouldn't we specify a full path to some intermediates directory here?

Reply:
Environment variables are not set and we need this file to revive them. That's why we can't use any of the intermediate dirs - we simply don't know them here.